### PR TITLE
[VnV Report] NFRs Evaluation: Usability and Humanity Requirements

### LINKED ISSUE #451: [VnV Report] Nonfunctional Requirements Evaluation - Usability and Humanity Requirements
## Content

- Functional Requirements Evaluation

## Rubric Sections

- Content
- Spelling and grammar and style
- VnV Report Matches VnV Plan
- Nonfunctional quality 1
- Nonfunctional quality 2

### PR DESCRIPTION
**Pull request by TimChoy**
Close #451

### CODE REVIEW

**Comment by joyxiao99** [comment]
add revision history to hazard analysis

File: docs/VnVReport/VnVReport.tex [hunk 1]
```diff
@@ -104,9 +106,33 @@ \section{Functional Requirements Evaluation}
 
 \section{Nonfunctional Requirements Evaluation}
 
-\subsection{Look and Feel Requirements}
+<<<<<<< HEAD
```
Comment (joyxiao99) [inline docs/VnVReport/VnVReport.tex]: remove

File: docs/VnVReport/VnVReport.tex [hunk 2]
```diff
@@ -104,9 +106,33 @@ \section{Functional Requirements Evaluation}
...
+\subsection{Usability and Humanity Requirements}
 \begin{enumerate}
+	\item \textbf{NFRT-UH1-1}
```
Comment (joyxiao99) [inline docs/VnVReport/VnVReport.tex]: can you add the other info here as well like "Type: Dynamic, Manual", etc.

Comment (LeonSo7) [reply docs/VnVReport/VnVReport.tex]: Arkin also includes "input" and "output". I've changed the other tests to be consistent.

File: docs/HazardAnalysis/HazardAnalysis.tex [hunk 3]
```diff
@@ -181,11 +181,11 @@ \section{Critical Assumptions}
 		 & \begin{enumerate}[label=\alph*., leftmargin=*]
 			   \item Loss of internet
 			   \item Inactive or stale connection
-			   \item Web socket failure
+			         %    \item Web socket failure
 		   \end{enumerate}
 		 & \begin{enumerate}[label=\alph*., leftmargin=*]
 			   \item Notify user of chat disconnection
```
Comment (LeonSo7) [inline docs/HazardAnalysis/HazardAnalysis.tex]:
```suggestion
			   \item Notify user of network/internet disconnection
```

File: docs/VnVReport/VnVReport.tex (hunk 2 above)
Comment (TimChoy) [reply docs/VnVReport/VnVReport.tex]: Some requirements do not have certain fields, looking at the VnV Plan, I've kept those consistent

**Review by joyxiao99 — APPROVED** [review]

**Pull request merged**